recipes/matchpy/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "matchpy" %}
+{% set version = "0.4.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8de067f7537988280891bfdc8df6f498b1f314a51de3fc41c87815fa95068ce6
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - hopcroftkarp >=1.2,<2.0
+    - multiset >=2.0,<3.0
```

Comment:
Are these 2 needed when building?

Reply:
Yeah, see the prior build failures. I get can't use setuptools downloading in conda build errors.

Reply:
These aren't in setup_requires

Reply:
These are in the install_requires. I had thought conda/conda-forge had some way to avoid putting the runtime dependencies in the `build` section these days, but it gave me the error when I built it, so I put them there.